src/main/resources/templates/progress/teamList.html

```diff
@@ -46,10 +46,13 @@ <h1>Týmy</h1>
                 <p>Žádný tým nenalezen<p/>
             </div>
 
-            <div th:each="team : ${teams}" class="cypher organizer">
```

Comment:
v tomto file na radku 42 a 45 je podminka <div th:if="${teams.empty and search == null}">
asi by to melo byt nahrazeno teamsProgress:
<div th:if="${teamsProgress.empty and search == null}">

Reply:
opraveno, bylo to ještě na řádku 8